src/app/GitUI/Editor/FileViewer.cs

```diff
@@ -786,7 +786,7 @@ public Task ViewFileAsync(string fileName, bool isSubmodule = false, FileStatusI
             string GetFileText()
             {
                 using FileStream stream = File.Open(fullPath, FileMode.Open, FileAccess.Read, FileShare.ReadWrite);
-                using StreamReader reader = new(stream, GitModule.LosslessEncoding);
+                using StreamReader reader = ICSharpCode.TextEditor.Util.FileReader.OpenStream(stream, GitModule.LosslessEncoding);
```

Comment:
code style if possible:
```suggestion
using ICSharpCode.TextEditor.Util;
                using StreamReader reader = FileReader.OpenStream(stream, GitModule.LosslessEncoding);
```

Reply:
done